package.json

```diff
@@ -15,9 +15,10 @@
   "scripts": {
     "postinstall": "npx run-s lerna:bootstrap lerna:install lerna:install:wasmfs lerna:link",
     "build": "npm run lerna:build",
-    "dev": "npm run lerna:build:wasm-transformer && npx parcel examples/wasm-shell/index.html",
+    "cli": "./packages/cli/bin/run",
+    "dev": "npx run-s lerna:build:wasm-transformer dev:serve",
+    "dev:serve": "parcel examples/wasm-shell/index.html",
```

Comment:
Small nit: when you are running a npm script, node/npm will automatically add the `node_modules/.bin` folder to the `PATH`, so you can execute `parcel` instead of `npx parcel` or `npm run parcel`.

However I'll use npx parcel to keep consistency with the project

Reply:
@syrusakbary Yep, but there are cases in which it doesn't work if I'm not mistaken? I'll do some research, perhaps its worth opening an issue to remove the proceeding `npx`? 😄 

But I agree consistency would be great! My apologies if the pattern ended up being unnecessary 😂